tdrs-frontend/src/components/Header/Header.test.js

```diff
@@ -45,9 +45,9 @@ describe('Header', () => {
         <Header />
       </Provider>
     )
-    const reportsLink = wrapper.find('#reports')
+    const reportsLink = wrapper.find('#data-files')
     expect(reportsLink).toExist()
-    expect(reportsLink).toIncludeText('Reports')
+    expect(reportsLink).toIncludeText('Data Files')
```

Comment:
```suggestion
    expect(dataFilesLink).toIncludeText('Data Files')
```
the line above this also should get changed if you take these suggestions.